scripts/get_es_numbers.py

```diff
@@ -1,15 +1,31 @@
 import requests
 from typing import Dict
+import click
 TYPES = ["organism", "specimen", "dataset", "experiment", "file"]
-ES_HOST = 'http://wp-np3-e2:9200'
 
-
-def main():
-    numbers: Dict[str, int] = read_number_from_es()
+@click.command()
+@click.option(
+    '--es_host',
+    default="http://wp-np3-e2:9200",
+    help='Specify the Elastic Search server (port should be included), default to be http://wp-np3-e2:9200.'
+)
+@click.option(
+    '--es_index_prefix',
+    default="faang_build",
+    help='Specify the Elastic Search index prefix, default to be faang_build.'
+         'Combined with serial (default 3) then the indices will be faang_build_1_organism etc.'
+)
+@click.option(
+    '--serial',
+    default=3,
+    help="Combined with es_index_prefix"
+)
+def main(es_host, es_index_prefix, serial):
```

Comment:
It would be great to add documentation to this script: what it does, which parameters it accepts and what it will return (print)...
So general documentation (that purpose of this script) should go at top of the file, and then you can add specific documentation for each function.

Reply:
documents added